---
layout: single
permalink: /publications/
title: "Publications"
author_profile: true
header:
    overlay_image: /assets/images/101516_hybrids_opener_free.JamesCarey.v5.png
    caption: "Photo credit: [James Carey](https://https://www.debutart.com/artist/james-carey)"
---
### Preprints

**Hsieh PH**, Dang V, Vollger MR, Mao Y, Huang T, Dishuck PC, Baker C, Cantsilieris S, Lewis AP, Munson KM, Sorensen M, Welch AE, Underwood JG, Eichler EE. (2020) Opposing selective forces operating on human-specific duplicated *TCAF* genes in Neanderthals and humans. (*In revision*) [Link](https://assets.researchsquare.com/files/rs-77798/v1_stamped.pdf){: .btn .btn--info .btn--small}
{: .notice--info}

Johansson PA, Brattås PL, Douse CH, **Hsieh PH**, Pontis J, Grassi D, Garza R, Jönsson ME, Atacho DAM, Pircs K, Eren F, Sharma Y, Johansson J, Trono D, Eichler EE, Jakobsson J. (2020) A human-specific structural variation at the *ZNF558* locus controls a gene regulatory network during forebrain development. *bioRxiv* 2020.08.18.255562 (*In revision*) [Link](https://doi.org/10.1101/2020.08.18.255562){: .btn .btn--info .btn--small}
{: .notice--info}

Wilfert AB, Turner TN, Murali SC, **Hsieh PH**, Sulovari A, Wang T, Coe BP, Guo H, Hoekzema K, Bakken TE, Winterkorn LH, Evani US, Byrska-Bishop M, Earl RK, Bernier RA, Zody MC, Eichler EE, SPARK Consortium. (2020) Recent ultra-rare inherited mutations identify novel autism candidate risk genes. *bioRxiv* 2020.02.10.932327 (In revision) [Link](https://doi.org/10.1101/2020.02.10.932327){: .btn .btn--info .btn--small}
{: .notice--info}


### Peer-Reviewed Publications

Logsdon GA, Vollger MR, **Hsieh PH**, Mao Y, Liskovykh MA, Koren S, Nurk S, Mercuri L, Dishuck PC, Rhie A, de Lima LG, Dvorkina T, Porubsky D, Mikheenko A, Bzikadze AV, Kremitzki M, Graves-Lindsay TA, Jain C, Hoekzema K, Murali SC, Munson KM, Baker C, Sorensen M, Lewis AM, Surti U, Gerton JL, Larionov V, Ventura M, Miga KH, Phillippy AM, Eichler EE. (2021) The structure, function, and evolution of a complete human chromosome 8. *Nature* (*Accepted*) [Link](https://www.biorxiv.org/content/10.1101/2020.09.08.285395v1.abstract){: .btn .btn--info .btn--small}
{: .notice--info}

Ebert P, Audano PA, Zhu Q, Rodriguez-Martin B, Porubsky D, Bonder MJ, Sulovari A, Ebler J, Zhou W, Mari RS, Yilmaz F, Zhao X, **Hsieh PH**, Lee J, Kumar S, Lin J, Rausch T, Chen Y, Ren J, Santamarina M, Höps W, Ashraf H, Chuang NT, Yang X, Munson KM, Lewis AP, Fairley S, Tallon LJ, Clarke WE, Basile AO, Byrska-Bishop M, Corvelo A, Chaisson MJP, Chen J, Li C, Brand H, Wenger AM, Ghareghani M, Harvey WT, Raeder B, Hasenfeld P, Regier A, Abel H, Hall I, Flicek P, Stegle O, Gerstein MB, Tubio JMC, Mu Z, Li YI, Shi X, Hastie AR, Ye K, Chong Z, Sanders AD, Zody MC, Talkowski ME, Mills RE, Devine SE, Lee C, Korbel JO, Marschall T, Eichler EE. (2021) Haplotype-resolved diverse human genomes and integrated analysis of structural variation. *Science* 366, eabf7117. DOI: 10.1126/science.abf7117 [Link](https://science.sciencemag.org/content/early/2021/02/24/science.abf7117){: .btn .btn--info .btn--small}
{: .notice--info}

Porubsky D, Sanders AD, Höps W, **Hsieh PH**, Sulovari A, Sorensen M, Murali SC, Gordon D, Cantsilieris S, Pollen AA, Antonacci F, Marschall T, Korbel JO, Eichler EE. (2020) Recurrent inversion toggling and great ape genome evolution. *Nat Genet.*;52(8):849-858. [Link](https://www.nature.com/articles/s41588-020-0646-x){: .btn .btn--info .btn--small}
{: .notice--info}

Cantsilieris S, Sunkin SM, Johnson ME, Anaclerio F, Huddleston J, Baker C, Dougherty ML, Underwood J, Sulovari A, **Hsieh PH**, Yafei M, Catacchio CR, Malig M, Welch A, Sorensen M, Munson K, Jiang W, Ventura M, Lamb BT, Conlon RA, Eichler EE. (2020) An evolutionary driver of interspersed segmental duplications in primates. *Genome Biol* 21, 202. [Link](https://doi.org/10.1186/s13059-020-02074-4){: .btn .btn--info .btn--small}
{: .notice--info}

**Hsieh PH**, Vollger MR, Dang V, Porubsky D, Baker C, Cantsilieris S, Hoekzema K, et al. (2019) Adaptive archaic introgression of copy number variants and the discovery of previously unknown human genes. *Science* 366, eaax2083. DOI: 10.1126/science.aax2083. [Link](https://science.sciencemag.org/content/366/6463/eaax2083){: .btn .btn--info .btn--small}
{: .notice--info}

Hallmark B, Karafet TM, **Hsieh PH**, Osipova LP, Watkins JC, Hammer MF. (2019) Genomic Evidence of Local Adaptation to Climate and Diet in Indigenous Siberians. *Mol Bio Evol* 36 (2), 315-327 [Link](https://pubmed.ncbi.nlm.nih.gov/30428071/){: .btn .btn--info .btn--small}
{: .notice--info}

Kronenberg ZN, Fiddes IT, Gordon D, Murali S, Cantsilieris S, Meyerson OS, Underwood JG, Nelson BJ, Chaisson MJP, Dougherty ML, Munson KM, Hastie AR, Diekhans M, Hormozdiari F, Lorusso N, Hoekzema K, Qiu R, Clark K, Raja A, Welch AE, Sorensen M, Baker C, Fulton RS, Armstrong J, Graves-Lindsay TA, Denli AM, Hoppe ER, **Hsieh PH**, Hill CM, Pang AWC, Lee J, Lam ET, Dutcher SK, Gage FH, Warren WC, Shendure J, Haussler D, Schneider VA, Cao H, Ventura M, Wilson RK, Paten B, Pollen A, and Eichler EE. (2018) High-resolution comparative analysis of great ape genomes. *Science* 360, eaar6343 [Link](https://pubmed.ncbi.nlm.nih.gov/29880660/){: .btn .btn--info .btn--small}
{: .notice--info}

Prufer K., de Filippo C, Grote S., Mafessoni F, Korlevic P, Hajdinjak M, Vernot B, Skov L, **Hsieh PH**, Peyregne S, et al. (2017). A high-coverage Neandertal genome from Vindija Cave in Croatia. *Science* 358, 655-658. [Link](https://pubmed.ncbi.nlm.nih.gov/28982794/){: .btn .btn--info .btn--small}
{: .notice--info}

**Hsieh PH**, Hallmark B, Watkins J, Karafet TM, Osipova LP, Gutenkunst RN, Hammer MF. (2017) Exome sequencing provides evidence of polygenic adaptation to a fat-rich animal diet in indigenous Siberian populations. *Mol Bio Evol* 34 (11), 2913-2926. [Link](https://academic.oup.com/mbe/article/34/11/2913/4098817){: .btn .btn--info .btn--small}
{: .notice--info}

Ragsdale AP, Coffman AJ, **Hsieh PH**, Struck TJ, Gutenkunst RN. (2016) Triallelic population genomics for inferring correlated fitness effects of same site nonsynonymous mutations. *Genetics* 203 (1), 513-523. [Link](https://pubmed.ncbi.nlm.nih.gov/27029732/){: .btn .btn--info .btn--small}
{: .notice--info}

**Hsieh PH**, Woerner AE, Lachance J, Tishkoff S, Wall JD, Gutenkunst RN, Hammer MF. (2016) Model-based analyses of whole-genome data reveal a complex evolutionary history involving archaic introgression in Central African Pygmies. *Genome Research* 26: 291. [Link](https://genome.cshlp.org/content/26/3/291.short){: .btn .btn--info .btn--small}
{: .notice--info}

**Hsieh PH**, Veeramah KR, Lachance J, Tishkoff S, Wall JD, Gutenkunst RN, Hammer MF. (2016) Whole genome sequence analyses of Western Central African Pygmy hunter-gatherers reveal a complex demographic history and identify candidate genes under positive natural selection. *Genome Research* 26: 279. [Link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772012/){: .btn .btn--info .btn--small}
{: .notice--info}

Coffman AJ, **Hsieh PH**, Gravel S, Gutenkunst RN. (2016) Computationally efficient composite likelihood statistics for population genetic inference. *Mol Bio Evol* 33:591. [Link](https://academic.oup.com/mbe/article/33/2/591/2579696){: .btn .btn--info .btn--small}
{: .notice--info}

Edwards T, Tollis M, **Hsieh PH**, Liu Z, Kusumi K, Culver M, Murphy RW. Assessing models of speciation under different biological scenarios; an empirical study using multi-locus and RNA-seq analyses. *Ecology and Evolution* 6:379 (2016) [Link](https://onlinelibrary.wiley.com/doi/abs/10.1002/ece3.1865){: .btn .btn--info .btn--small}
{: .notice--info}

**Hsieh PH**, Chen IY, Lin YT, Kuo SY. (2004) An XOR-based Reed-Solomon Algorithm for Advanced RAID Systems, *Proceedings of the 19th IEEE International Symposium on Defect and Fault Tolerance in VLSI Systems* (DFT'04), Cannes, France. [Link](https://ieeexplore.ieee.org/abstract/document/1347837){: .btn .btn--info .btn--small}
{: .notice--info}

